recipes/write_behind/README.md

```diff
@@ -0,0 +1,99 @@
+# RedisGears Write-Behind Recipe
+
+## System requirements
+
+* Redis v5.0.7 or above,
+* Relational database: MySQL (any version will do) or Oracle (tested with 11g and 12c) database,
+* Redis Enterprise Software v5.4.11-2 and above,
+* RedisGears module build for Ubuntu Bionic
+
+## Configuration
+
+TBD: key names and tables
+
+## Installing on Redis Enterprise Software host(s)
```

Comment:
You do not need RedisEnterprise to run it, you can run it with vanila Redis, So I think we do not need this part..

Reply:
You don't need it, but when you need to install it on Redis Enterprise, it specifies how to do it.